include/oneapi/dpl/pstl/execution_defs.h

```diff
@@ -184,10 +116,6 @@ template <class _ExecPolicy, class _T = void>
 using __enable_if_host_execution_policy =
```

Comment:
Why do we need this trait? Is it necessary beyond for_loop?

Reply:
Fixed